ci/verify_examples.sh

```diff
@@ -6,13 +6,25 @@ FAILED=()
 SRCDIR="${SRCDIR:-$(pwd)}"
 WARNINGS=()
 
+# Sandboxes listed here should be regarded as broken(/untested) until whatever
```

Comment:
These were also broken in 1.26?

Reply:
actually not `double-proxy` - just the other 2 ill remove now ...

Reply:
updated